NSS-Day-Cohort-53/giffygram-rantingtyras
# added delete post functionality

```text
#### Changes Made
1. Modified file `postList.js` to include `delete post button` and deletes post.
2. modified provider js, fixed the fetch call  url
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout iw-deletePost
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > if you are viewing a post that was posted to the currently logged in user
    >then you will be given a delete button by all of your posts
    > When clicked, will delete that post and re render the page
    > delete button will not populate for posts that don't belong to the currently logged in user
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
